app/uk/gov/hmrc/tai/viewModels/incomeTaxHistory/IncomeTaxHistoryViewModel.scala

```diff
@@ -22,7 +22,7 @@ case class IncomeTaxHistoryViewModel(
   employerName: String,
   ern: String,
   startDate: LocalDate,
-  endDate: LocalDate,
+  endDate: Option[LocalDate],
```

Comment:
all the other optional fields are called maybe, so I would call it maybeEndDate rather than endDate

Reply:
👍